server/cluster/table_manager_test.go

```diff
@@ -39,6 +39,7 @@ const (
 	testRootPath                    = "/rootPath"
 	defaultIDAllocatorStep          = 20
 	defaultThreadNum                = 20
+	defaultTablePartitionNum        = 2
```

Comment:
Yes, it is a cluster option, only the unit test here uses a default value.

Reply:
I guess different tables should have different partition number which is configured by the user's create table request.

Reply:
User specifies the number of partitions in the partition table through SQL like this:
```
CREATE TABLE `demo` (`name` string TAG, `value` double NOT NULL, `t` timestamp NOT NULL, TIMESTAMP KEY(t)) PARTITION BY KEY(name) PARTITIONS 2 ENGINE=Analytic with (enable_ttl="false")
```

`TablePartitionNum` is not the number of partitions in the partition table, but the number of routing nodes in the partition, which is transparent to users.

Reply:
The name is confusing...

Maybe this should not be configured as an option of cluster? It can just be a default value, and I guess a better way is to define a ratio based on the total nodes telling us to choose how many routing nodes to select for partition table.